docs/DeveloperGuide.md

```diff
@@ -13,6 +13,44 @@ Project team leaders can:
 * see an estimate of how much workload each member has.
 * receive information regarding upcoming deadlines.
 
+---
+
+## User stories
+
+####Priorities: High (must have) - ``* * *``, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
```

Comment:
Add a space between ### and Priorities.

Reply:
Also, suggest turning this into an unordered list but maybe a bit redundant;
- High ...
- Medium ...
- Low ...